Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -216,7 +216,8 @@ class BrowserViewController: UIViewController {
             updateURLBarDisplayURL(tab)
             navigationToolbar.updateBackStatus(webView.canGoBack)
             navigationToolbar.updateForwardStatus(webView.canGoForward)
-            navigationToolbar.updateReloadStatus(tab.loading)
+            urlBar.locationView.loading = tab.loading
+            // TODO: Update reload status on TabLocationView
```

Comment:
nit: `TODO:` is this intentional for future work and if so, can you please check we have a ticket to cover this work

Reply:
Nope I just used the TODO's were only for my own sake during the PR. Just accidentally left this one here, will remove

Reply:
👍